CONTRIBUTING.md

```diff
@@ -38,3 +38,25 @@ We’ll do our best to review your pull request (or “PR” for short) quickly.
 Each PR should, as much as possible, address just one issue and be self-contained.
 Smaller the set of changes in the pull request is, the quicker it can be reviewed and
 merged - if you have ten small, unrelated changes, then go ahead and submit ten PRs.
+
+# How to release
+
+- Be sure to be rebased on `master` branch
+- Edit `osc_sdk/sdk.py` and update `SDK_VERSION` following [semantic versioning 2.0.0](https://semver.org/)
+- Update `setup.py` to update `version`
+- Run all tests with `make test`
+- Build with `make build` and check `dist` folder
+- Go to pkg and run `make`. Make sure you have `osc-cli-x86_64.zip` and `osc-cli-x86_64.AppImage`
+- Commit release update: `git commit -am "osc-cli v1.x.x"`
+- Tag new release `git tag v1.x.x`
+- Push release to new branch `git push --tags outscale master:v1.x.x`
+- Create PR to run github actions and get validation
+- Make the PR reviewed
+- Create release page
+- Add major change to release page
+- Upload packages in release page
```

Comment:
- Upload `osc-cli-x86_64.zip` and `osc-cli-x86_64.AppImage` packages in release page

Reply:
Yes, better than implicit.
We can also ad `dist/osc_sdk-1.6-py2.py3-none-any.whl` and `dist/osc-sdk-1.6.tar.gz`.